src/sw/redis++/queued_redis.hpp

```diff
@@ -95,11 +95,11 @@ auto QueuedRedis<Impl>::command(Input first, Input last)
         throw Error("command: empty range");
     }
 
-    auto cmd = [](Connection &connection, Input first, Input last) {
+    auto cmd = [](Connection &connection, Input lfirst, Input llast) {
```

Comment:
Rename `lfirst` as `start` and rename `llast` as `stop`. Please also modify other `lfirst` and `llast` variables.